src/validations.ts

```diff
@@ -26,6 +38,18 @@ export function validateRequiredField(
   return null;
 }
 
+export function validateMinLength(
```

Comment:
What do you think about adding some tests for this validation?

Reply:
yes i need put some